nix/nixosModule.nix

```
@@ -0,0 +1,96 @@
+{
+  inputs,
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+with lib;
+let
+  cfg = config.programs.ignis;
+in
+{
+  options.programs.ignis = {
+    enable = mkEnableOption "Enable the Ignis widget framework.";
```

I think there is should be the general description convention.
```suggestion
    enable = mkEnableOption "Enable the Ignis widget framework";
```